### PR TITLE
Add Ampli CLI configure command docs

### PR DESCRIPTION
**Pull request by bohan-amplitude**
# Amplitude Developer Docs PR

## Description

Add configure descriptions to `pull` step instruction
Add --deprecated-runtimes flag and example to `pull` command
Add `configure` command documentation

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs
@caseyamp

### CODE REVIEW

File: docs/data/ampli/cli.md [hunk 1]
```diff
@@ -243,6 +245,22 @@ USAGE
   $ ampli help [command]
 ```
 
+### `ampli configure`
+
+Configure and update source runtime
```
Comment (caseyamp) [inline docs/data/ampli/cli.md:250]:
```suggestion
Configure and update source runtime.
```

**Review by caseyamp — APPROVED** [review]

File: docs/data/ampli/cli.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ Now that you're done setting Ampli up, you’re ready to generate your analytics
 ampli pull {source-name}
 ```
 
-The pull command takes one argument: the name of a source created in your Amplitude Data account. For example, a source might be `ios`, `android`, `web`, or `backend`. Events associated with a particular source appear in the autogenerated SDK. Those that aren't don't. This makes sure Amplitude tracks your events consistently across all sources you track them on.
+The pull command takes one argument: the name of a source created in your Amplitude Data account. For example, a source might be `ios`, `android`, `web`, or `backend`. Follow the prompt options to configure a newly created sources. Events associated with a particular source appear in the autogenerated SDK. Those that aren't don't. This makes sure Amplitude tracks your events consistently across all sources you track them on.
```
Comment (justin-fiedler) [inline docs/data/ampli/cli.md:57]: nit: can we please update this to call out the arg is optional? I would like to educate users to run `ampli pull` without any arguments and instead rely on the prompts to guide the process.
cc @caseyamp 
```suggestion
The pull command takes one (optional) argument: the name of a source created in your Amplitude Data account. For example, a source might be `ios`, `android`, `web`, or `backend`. Follow the prompt options to configure a newly created sources. Events associated with a particular source appear in the autogenerated SDK. Those that aren't don't. This makes sure Amplitude tracks your events consistently across all sources you track them on.
```

**Review by justin-fiedler — APPROVED** [review]

File: docs/data/ampli/cli.md (hunk 2 above)
Comment (caseyamp) [reply docs/data/ampli/cli.md:57]: I think it might flow better to say "the pull command takes one optional argument", but I'm okay with this.

**Review by qingzhuozhen — COMMENTED** [review]
Do we need to mention the starting version the new command is available?

**Review by justin-fiedler — CHANGES_REQUESTED** [review]
Add note about min version.

**Comment by justin-fiedler** [comment]
Closing forked PR. See final implementation here
https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/pull/520

**Pull request closed without merging**